.github/workflows/ai-validation.yml

```
@@ -0,0 +1,166 @@
+            echo "" >> $GITHUB_STEP_SUMMARY
+            echo "For detailed results, visit the [AI Validation Dashboard](${APP_URL}/admin/ai/validation-pipeline)" >> $GITHUB_STEP_SUMMARY
+          else
+            echo "⚠️ **Validation process encountered issues**" >> $GITHUB_STEP_SUMMARY
+            echo "" >> $GITHUB_STEP_SUMMARY
+            echo "Please check the validation pipeline manually to verify model accuracy." >> $GITHUB_STEP_SUMMARY
+            echo "You can access the [AI Validation Dashboard](${APP_URL}/admin/ai/validation-pipeline) for more details." >> $GITHUB_STEP_SUMMARY
+          fi
+
+      - name: Send notification on validation issues
+        if: steps.results.outputs.success == 'true' && env.PASS_RATE < 85
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)